src/main/kotlin/org/serenityos/jakt/folding/JaktFoldingBuilder.kt

```diff
@@ -0,0 +1,51 @@
+package org.serenityos.jakt.folding
+
+import com.intellij.lang.ASTNode
+import com.intellij.lang.folding.FoldingBuilderEx
+import com.intellij.lang.folding.FoldingDescriptor
+import com.intellij.openapi.editor.Document
+import com.intellij.openapi.editor.FoldingGroup
+import com.intellij.openapi.project.DumbAware
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import com.intellij.refactoring.suggested.endOffset
+import com.intellij.refactoring.suggested.startOffset
+import org.serenityos.jakt.psi.declaration.JaktImportMixin
+import org.serenityos.jakt.psi.findChildrenOfType
+
+class JaktFoldingBuilder : FoldingBuilderEx() {
```

Comment:
I think we should inherit from `CustomFoldingBuilder` like the other folding class we have. The APIs seem pretty much the same, just mostly different names.